test/unit/YieldTokenIncidents/redeemPayout.js

```diff
@@ -709,4 +770,159 @@ describe('redeemPayout', function () {
       .to.emit(yieldTokenIncidents, 'IncidentPayoutRedeemed')
       .withArgs(coverOwner2.address, parseEther('90'), 1, 2);
   });
+
+  it('reverts if system is paused', async function () {
+    const { yieldTokenIncidents, cover, assessment, ybEth, master } = this.contracts;
+    const [coverOwner1] = this.accounts.members;
+    const [advisoryBoard] = this.accounts.advisoryBoardMembers;
+    const { timestamp } = await ethers.provider.getBlock('latest');
+    const segment = await getCoverSegment();
+    await cover.createMockCover(
+      coverOwner1.address,
+      2, // productId using ybEth
+      ASSET.ETH,
+      [segment],
+    );
+
+    await yieldTokenIncidents
+      .connect(advisoryBoard)
+      .submitIncident(2, parseEther('1'), timestamp + 2, parseEther('100'), '');
+
+    await assessment.connect(advisoryBoard).castVote(0, true, parseEther('100'));
+
+    {
+      const { payoutCooldownInDays } = await assessment.config();
+      const { end } = await assessment.getPoll(0);
+      await setTime(end + daysToSeconds(payoutCooldownInDays));
+    }
+
+    await master.pause();
+
+    await ybEth.connect(coverOwner1).approve(yieldTokenIncidents.address, parseEther('10000'));
+    await expect(
+      yieldTokenIncidents.connect(coverOwner1).redeemPayout(0, 0, 0, parseEther('100'), coverOwner1.address, []),
+    ).to.be.revertedWith('System is paused');
+  });
+
+  it('reverts if caller is not a member', async function () {
+    const { yieldTokenIncidents, cover, assessment, ybEth } = this.contracts;
+    const [coverOwner1, coverOwner2] = this.accounts.members;
+    const [advisoryBoard] = this.accounts.advisoryBoardMembers;
+
+    const { timestamp } = await ethers.provider.getBlock('latest');
+    const segment = await getCoverSegment();
+    await cover.createMockCover(
+      coverOwner1.address,
+      2, // productId using ybEth
+      ASSET.ETH,
+      [segment],
+    );
+
+    await yieldTokenIncidents
+      .connect(advisoryBoard)
+      .submitIncident(2, parseEther('1'), timestamp + 2, parseEther('100'), '');
+
+    await assessment.connect(advisoryBoard).castVote(0, true, parseEther('100'));
+
+    {
+      const { payoutCooldownInDays } = await assessment.config();
+      const { end } = await assessment.getPoll(0);
+      await setTime(end + daysToSeconds(payoutCooldownInDays));
+    }
+
+    await ybEth.connect(coverOwner1).approve(yieldTokenIncidents.address, parseEther('10000'));
+    await expect(
+      yieldTokenIncidents.connect(coverOwner2).redeemPayout(0, 0, 0, parseEther('100'), coverOwner1.address, []),
+    ).to.be.revertedWith('Only the cover owner or approved addresses can redeem');
```

Comment:
the assertion here checks that it's different owner, not that it's not a member like the text of the test states.

Let's fix this by changing the address that calls the `redeemPayout` function